### PR TITLE
My work and About me features

### PR DESCRIPTION
**Pull request by Rebelzob**
# Portfolio 

- Added My Work HTML and CSS
- Added About Me HTML and CSS
- Applied functions to the buttons

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,43 +28,237 @@
...
+          </ul>
+        </div>
+        <div class="ab-bt-blocks">
+          <ul>
+            <h4 class="ab-titles skills"> Skills</h4>
+            <li class="about-bullet">Database Management</li>
+            <li class="about-bullet">Version Control</li>
+            <li class="about-bullet">CLI</li>
+            <li class="about-bullet">API Design</li>
+            <li class="about-bullet">Web Development</li>
```
Comment (Owusu-Desmond) [inline index.html:255]: - [ ] Please make sure that you adjust the spacing interval between the listed skills of this section to match the Figma design 

  ### YOUR DESIGN
  image
  
  ### FIGMA DESIGN
  image

File: styles.css [hunk 2]
```diff
@@ -89,4 +100,318 @@
...
+
+  /* Cards section */
+
+  .cards {
+    background-color: #e5e5e5;
+    display: grid;
+    text-shadow: 1px 1px 2px ccc;
+    padding-left: 24px;
+    padding-bottom: 24px;
+  }
```
Comment (Owusu-Desmond) [inline styles.css:196]: - [ ] Please make sure that you give your project cards margin right to look exactly like the Figma design. Currently, You have no margin right for your project cards :blush: 
  
  ### Your design
  image
  
  ### Figma design 
  image

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @Rebelzob , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Rebelzob) [reply index.html:255]: @Owusu-Desmond thanks for your review!!

File: styles.css (hunk 2 above)
Comment (Tufahel) [inline styles.css:196]: - [ ] As per the previous reviewer's suggestion you need to add some margin to the right side of the cards to match the given design. You can add `margin-right: ` here to see the effect and adjust it.
- image

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @Rebelzob :wave:  

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️ Descriptive pull request.
✔️ Linter checks are passing.
✔️ Follow the correct GitHub flow.
✔️ Commit messages are meaningful.

## Required Changes ♻️

***Please kindly check the comments/review below.***

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 2 above)
Comment (Rebelzob) [reply styles.css]: thanks for the review, I misunderstood that particular change the previous reviewer was pointing out.

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🥳

Hi @Rebelzob 🙋‍♀️

Great work implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great, perfect!
-  ✅ Great work using the right GitHub Flow 
-  ✅ Your README looks good, well done!
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] You have a very good README 🥇Well done. To enhance the professionalism of the README, it would be best to remove the highlighted text, as it is intended as a placeholder and not meant to be included in the final version of the README. ✨💪

  image

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Rebelzob** [comment]
@KanzaTahreem Thank you for your review!

**Pull request merged**